docs/src/pages/Tutorial/tutorial.md

```diff
@@ -101,7 +101,7 @@ Create this html file:
       href="https://hyperapp.dev/tutorial-assets/style.css"
     />
     <script type="module">
-      import { h, app } from "https://unpkg.com/hyperapp"
+      import { h, app } from "https://unpkg.com/hyperapp";
```

Comment:
@zaceno Semicolons are back in 2020? 😉

Reply:
Oh rats... 😩  I forgot. I had got rid .prettierrc so the moment I saved my changes all the delightful semicolons came right back. 🤦‍♂ 

Ok hang on... I'll fix it

Reply:
@jorgebucaran done. Sorry bout that